packages/gafl-webapp-service/src/handlers/__tests__/attribution-handler.spec.js

```diff
@@ -9,7 +10,14 @@ jest.mock('../../constants', () => ({
     SOURCE: 'utmsource',
     TERM: 'utmterm'
   },
-  ATTRIBUTION_REDIRECT_DEFAULT: '/attribution/redirect/default'
+  ATTRIBUTION_REDIRECT_DEFAULT: '/attribution/redirect/default',
+  reference: {
+    reference: 'reference'
+  }
+}))
+
+jest.mock('../../uri', () => ({
+  RENEWAL_BASE: '/renew'
```

Comment:
This needs to change to something other than '/renew', so that we check we don't have anything hard-coded to the value rather than using the RENEWAL_BASE constant. Make it '/licence-renew-url' or something like that.